eclair-core/src/test/scala/fr/acinq/eclair/router/BaseRouterSpec.scala

```diff
@@ -103,7 +103,8 @@ abstract class BaseRouterSpec extends TestkitBaseClass {
       // let's set up the router
       val peerConnection = TestProbe()
       val watcher = TestProbe()
-      val nodeParams = Alice.nodeParams
+      import com.softwaremill.quicklens._
+      val nodeParams = Alice.nodeParams.modify(_.routerConf.routerBroadcastInterval).setTo(1 day) // "disable" auto rebroadcast
```

Comment:
Why is that change done here?

Reply:
I don't know if tests run faster or in a different order with scala 2.13/akka 2.6, but a race condition appeared.

Reply:
Damn, faster tests, I'm all in for that! But when will I have time for coffee breaks now?